Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM mcr.microsoft.com/playwright:v1.35.1-focal AS container
+FROM mcr.microsoft.com/playwright:v1.40.0-jammy AS container
```

Comment:
We can use `v1.40.1-jammy`

Reply:
Ah good one, even in their docs they have 1.40.0 all over the place. Where did you find it?

Reply:
`crane ls mcr.microsoft.com/playwright`